doc/versions.json

```diff
@@ -20,8 +21,17 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.6.99",
-      "nrfxlib": "1.6.99",
+      "ncs": "1.7.0",
+      "nrfxlib": "1.7.0",
+      "zephyr": "2.6.99",
+      "mcuboot": "1.7.99",
+      "nrfx": "2.5",
+      "tfm": "1.3.99",
+      "matter": ""
```

Comment:
@carlescufi , @tejlmand , kindly please confirm versions for 1.7.0...

Reply:
Yes, except matter which needs comment from @LuDuda

Reply:
As discussed internally, let's not specify any versioning for now.